README.md

```diff
@@ -23,123 +21,155 @@ This tool will eventually place the existing `puppet doc` command once feature p
 [contributing]: https://github.com/puppetlabs/puppet-strings/blob/master/CONTRIBUTING.md
 [committers]: https://github.com/puppetlabs/puppet-strings/blob/master/COMMITTERS.md
 
-Requirements
-------------
+## Installing Puppet Strings
+
+### Requirements
 
-In order to run strings you need to have the following software installed:
+To run Strings, you need the following software:
 
   * Ruby 1.9.3 or newer
   * Puppet 3.7 or newer
   * The `yard` Ruby gem
 
-Note that a few extra steps are necessary to install puppet-strings with Puppet Enterprise 3.8.
+Note that if you are running PE 3.8, you'll have a few extra steps to install puppet-strings.
 
-Installing the YARD Gem
------------------------
+### 1. Install the YARD Gem
 
 The easiest way to install the `yard` gem is with Puppet itself:
 
 For Puppet 4.x and Puppet Enterprise 2015.2 and later:
+
 ```
 $ puppet resource package yard provider=puppet_gem
 ```
 
 For Puppet 3.x:
+
 ```
 $ puppet resource package yard provider=gem
 ```
 
 For Puppet Enterprise 3.8:
+
 ```
 GEM_HOME=/opt/puppet/lib/ruby/gems/1.9.1 puppet resource package yard provider=gem
 ```
```

Comment:
I have no idea! And @er0ck , your question applies to the whole doc, right? @whopper ?